types/node/fs.d.ts

```diff
@@ -4261,7 +4261,7 @@ declare module "fs" {
      */
     export function cpSync(source: string | URL, destination: string | URL, opts?: CopySyncOptions): void;
 
-    interface GlobOptionsBase {
+    export interface GlobOptions<T extends Dirent | string = Dirent | string> {
```

Comment:
Why do the type parameter instead of the old way with the duplicated def?

Reply:
Arbitrary choice – one less symbol. I have no strong feelings either way.

Reply:
When it comes to type parameters, I'm wary of adding them just because they can never be removed as someone could instantiate this type and then we're stuck with it. The previous method seemed fine enough if it still works (but if it doesn't that's another story).

Reply:
I guess an unexported interface works too.